web-marketplace/src/components/organisms/DashboardNavigation/DashboardNavigation.Header.tsx

```diff
@@ -24,6 +24,11 @@ type Props = DashboardNavHeaderData & {
   onViewProfileClick?: (href: string) => void;
   hasWalletAddress?: boolean;
   wallet?: String;
+  hasOrganization?: boolean;
+  onCreateOrganization?: () => void;
+  unfinalizedOrgCreation?: boolean;
+  unfinalizedOrgName?: string;
+  onFinishOrgCreation?: () => void;
```

Comment:
I think both `onCreateOrganization` and `onFinishOrgCreation` could be required
this would avoid all these `?.()`